Add plugin: Image Border Style

shenoy-anurag opened the pull request:
# I am submitting a new Community Plugin
## Repo URL

Link to my plugin:
https://github.com/shenoy-anurag/obsidian-image-border-style

## Release Checklist
I have tested the plugin on
- [ ] Windows
- [x] macOS
- [ ] Linux
- [ ] Android (if applicable)
- [ ] iOS (if applicable)

My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
- [x] main.js
- [x] manifest.json
- [x] styles.css (optional)
- [x] GitHub release name matches the exact version number specified in my manifest.json (Note: Use the exact version number, don't include a prefix v)
- [x] The id in my manifest.json matches the id in the community-plugins.json file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
I have given proper attribution to these other projects in my README.md.

joethei commented:
[// const APP_TITLE = this.plugin.manifest.name + " " + this.plugin.manifest.version](https://github.com/shenoy-anurag/obsidian-image-border-style/blob/0ca68e484fe5720e56d4b6632c8124e5282c09af/main.ts#L148-L150)
Remove this code, it should never be used, see: [don't add a top-level heading in the settings tab, such as "General", "Settings", or the name of your plugin.](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Only+use+headings+under+settings+if+you+have+more+than+one+section.)

[.setName("Rounded Borders")](https://github.com/shenoy-anurag/obsidian-image-border-style/blob/0ca68e484fe5720e56d4b6632c8124e5282c09af/main.ts#L153)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

shenoy-anurag commented:
Hi Joethei,

1. I removed the commented code for a top-level heading.
2. Changed all UI text to sentence case.

Let me know if any other changes are required, thanks.
